internal/cli/bumpver.go

```diff
@@ -0,0 +1,148 @@
+package cli
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+
+	"github.com/futurice/jalapeno/internal/cli/option"
+	"github.com/futurice/jalapeno/pkg/recipe"
+	"github.com/futurice/jalapeno/pkg/ui/changelog"
+	"github.com/spf13/cobra"
+	"golang.org/x/mod/semver"
+)
+
+type bumpVerOpts struct {
+	RecipeName    string
+	RecipeVersion string
+	ChangelogMsg  string
+	option.Common
+	option.WorkingDirectory
+}
+
+func NewBumpVerCmd() *cobra.Command {
+	var opts bumpVerOpts
+	var cmd = &cobra.Command{
+		Use:   "bumpver",
+		Short: "Bump version number for recipe",
+		Long:  "Bump version number for recipe. By default prompts user for update increment (patch/minor/major) and changelog messsage. These can also be specified directly with the -v and -m flags.",
+		Args:  cobra.NoArgs,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return option.Parse(&opts)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := runBumpVer(cmd, opts)
+			return errorHandler(cmd, err)
+		},
+		PostRun: func(cmd *cobra.Command, args []string) {
+			cmd.Root().SetContext(cmd.Context())
+		},
+		Example: `# Prompt version increment and changelog message
+jalapeno bumpver
+		
+# Directly specify version and message
+jalapeno bumpver -v v1.0.0 -m "Hello world"`,
+	}
+
+	cmd.Flags().StringVarP(&opts.RecipeName, "recipe", "r", "", "Name of the recipe to upgrade")
```

Comment:
How the `RecipeName` [is used here](https://github.com/futurice/jalapeno/pull/140/files#diff-dc1f6e1c37681177c25d2a8b14e77a07b9a78c411d3cd0546a38006a9e6ed1aaR67) is actually a path, not just the name. Providing the recipe path could work similarly [as in `test` command](https://github.com/futurice/jalapeno/blob/main/internal/cli/test.go#L29-L33) as an argument.